components/normalizer/benches/composing_normalizer_nfc.rs

```diff
@@ -0,0 +1,40 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use criterion::{black_box, BenchmarkId, Criterion, Throughput};
+
+use icu_normalizer::ComposingNormalizer;
+
+fn function_under_bench(normalizer: &ComposingNormalizer, text: &str) {
+    normalizer.normalize(text);
+}
+
+pub fn criterion_benchmark(criterion: &mut Criterion) {
+    let group_name = "composing_normalizer_nfc";
+    let normalizer: ComposingNormalizer =
+        ComposingNormalizer::try_new_nfc_unstable(&icu_testdata::unstable()).unwrap();
+    let params = black_box([
+        "a\u{0308}",
```

Comment:
For the operations that take `&str`, I think we should bench natural-language content in a handful of languages that exercise interesting things. The main problem is obtaining license-wise unencumbered content. (Also, it's hard to decide how long the test strings should be. They should be longer than the strings here. Especially for English, we should also have a benchmark where the input spans multiple memory pages.)

Reply:
should we use files from https://github.com/unicode-org/icu/tree/main/icu4j/perf-tests/data/collation ?

Reply:
Some of the `TestRandomWordsUDHR_` seem suitable for normalization testing, but they don't appear to cover all the languages suggested in #2444. It would be particularly relevant to have Vietnamese (frequent multidiacritic characters) and Korean (special case unlike any other script/language) test data.

Reply:
I've pulled all of them and added them to normalization bench